index.js

```diff
@@ -6,7 +6,8 @@ try{
     Proxy.create({}, handler);
 }catch(e){
     // hack to enable the harmony-proxies flag by default
-    require("harmonize")(["harmony-proxies"]);
+    require("harmonize")(["harmony", "harmony-proxies", "harmony_proxies"]);
+    require('harmony-proxy');
```

Comment:
What's the motivation for using both packages?

Reply:
oh, this is my bad attempt. it is only need harmonize@2.0.0
It works after update 👍 
https://travis-ci.org/vntk/vntk-conlleval/builds/323469196